src/components/03-components/06-dashboard/16-tile/tile.njk

```diff
@@ -1,19 +1,38 @@
-<h3 class="margin-top-0">Basic Tile Example</h3>
-<div class="sds-tile">
-  <a href="#" class="sds-tile__content">
-    {% render '@layered-icon', tiles[0].icon %}
-    <div>
-      <div class="sds-tile__title">Tile Title</div>
-      <div class="sds-tile__details">tile details</div>
+<div class="grid-row flex-align-center">
```

Comment:
Is there a reason the same tile is being displayed three times?

Reply:
yep, to show how they behave in a row from desktop to mobile